scripts/env.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if (( $# != 1 )); then
+    echo "Usage: source env.sh aws_profile_name"
+    echo "       where aws_profile_name is an AWS CLI named profile"
+    echo "       https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html"
+    exit 1
+else
+    export AWS_PROFILE=$1
+
+    AWS_ACCESS_KEY_ID=$(aws configure get aws_access_key_id --profile "$AWS_PROFILE")
+    AWS_SECRET_ACCESS_KEY=$(aws configure get aws_secret_access_key --profile "$AWS_PROFILE")
+    AWS_REGION=$(aws configure get region --profile "$AWS_PROFILE" || echo "$AWS_DEFAULT_REGION")
+    AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
```

Comment:
I'm setting Short-term access keys via Kion, and can't retrieve the account ID unless AWS_SESSION_TOKEN is available. Without it, the `source scripts/env.sh default` step returns:

`An error occurred (InvalidClientTokenId) when calling the GetCallerIdentity operation: The security token included in the request is invalid.`

Reply:
Oh, create long-term access keys. I think it's just a different tab in the Kion UI.

Reply:
I went crazy and read the Earthdata wiki documentation on Kion, and it says:

```
Short-term keys are the preferred type, as their short lifespan makes them better for security.  Kion currently expires short-term keys after 8 hours (420 minutes) following the key's generation.  

Long-term keys are generally used for build and deployment jobs in CI systems, such as Jenkins or Bamboo. Kion currently expires long-term keys after ninety (90) days with a warning via email ten (10) days prior to expiration.  
```

I'm lazy and happy to generate a key that doesn't have to be recreated every day, but seems like we should support the short-term key option, too.

Reply:
The AWS CLI docs seem to always refer to LTAKs, and Ops will only use LTAKs. I'm not sure if there's an immediate use-case for STAKs, but if there is, yeah, we could add that.